ssn/rdf/examples/Beer-SampleCollections-IBS-TH2.ttl

```diff
@@ -0,0 +1,36 @@
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd:  <http://www.w3.org/2001/XMLSchema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix qudt: <http://qudt.org/schema/qudt/> .
+@prefix schema: <http://schema.org/> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix sosa-env: <http://www.w3.org/ns/sosa/system-environment-properties#> .
+@prefix sosa: <http://www.w3.org/ns/sosa/> .
+@prefix ex: <http://example.org/> .
+@prefix sensor: <https://example.org/sensor/> .
+@base <http://example.org/data/> .
+
+<ACMEqualityAssuranceProgram> a prov:Activity, sosa:Sampling .
+
+<beerSamplingProcedure> a sosa:SamplingProcedure;
+    rdfs:label "The procedure used by employees when taking a sample of the beer during production and packaging." .
+
+<10/PO202402/21/0001/acmePorterSixPackLabSample> a sosa:Sample;
+    rdfs:label "A sample of beer within the six pack"@en ;
+    sosa:isResultOf <ACMEqualityAssuranceProgram>;
```

Comment:
`sosa:isResultOf` is meant to be used with `sosa:Execution` as range. Therefore `ACMEqualityAssuranceProgram` *should* be a `sosa:Sampling`. 

I think it is best if the examples keep to the spirit of the spec. While this creative use of predicates is not illegal it adds unnecessary entropy in my view.

Reply:
OK.